needed since we need libucontext for ppc, and other reasons (musl doesn't have
ucontext api, so the lib is external)
--
--- a/src/unwind/libunwind.pc.in
+++ b/src/unwind/libunwind.pc.in
@@ -6,6 +6,6 @@
 Name: libunwind
 Description: libunwind base library
 Version: @VERSION@
-Libs: -L${libdir} -lunwind
+Libs: -L${libdir} -lunwind -lucontext
 Libs.private: @LIBLZMA@ @LIBZ@
 Cflags: -I${includedir}
